native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -173,13 +173,23 @@ const themeMappings = {
   text: {
     color: {
       primary: 'buttonPrimaryTextColor',
+      secondary: 'buttonSecondaryTextColor',
+    },
+  },
+  button: {
+    borderColor: {
+      secondary: 'buttonSecondaryBorderColor',
     },
   },
   gradient: {
     primary: {
       startColor: 'buttonPrimaryGradientStartColor',
       endColor: 'buttonPrimaryGradientEndColor',
     },
+    secondary: {
+      startColor: 'buttonSecondaryBackgroundColor',
+      endColor: 'buttonSecondaryBackgroundColor',
```

Comment:
I understand the convenience here - ideally we wouldn't wrap the button in a linear gradient component if we arent't using it. Perhaps we can revisit this in a later PR (especially when we remove selected state etc).

Reply:
Yeah that makes sense, it's a bit of a hack having a non-changing gradient.